src/components/NoSearchResults.tsx

```diff
@@ -0,0 +1,34 @@
+import SearchIcon from '@mui/icons-material/Search';
+import { Box, Typography } from '@mui/material';
+import { ReactNode } from 'react';
+import { Trans } from 'react-i18next';
+
+interface NoSearchResultsProps {
+  children: ReactNode;
+}
+
+export const NoSearchResults = ({ children }: NoSearchResultsProps) => {
+  return (
+    <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          gap: '0.5rem',
+          mt: '2rem',
+          p: '1rem',
+        }}>
```

Comment:
Ved første øyekast var det litt vanskelig å se hva som er ansvaret/formålet med disse nå syns jeg 🤔 
Men forstår det som at den første boksen har ansvaret for plassering av innholdet, mens den andre bare har ansvaret for output fra Trans (hvertfall det jeg syns gir mest mening). Da er det kanskje ryddigere å flytte plasseringsting til den første, sånn at hver boks har bare ett ansvar?


```suggestion
    <Box sx={{ display: 'flex', justifyContent: 'center', m: '2rem 0.5rem' }}>
      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
```